paasta_tools/marathon_tools.py

```diff
@@ -204,33 +203,41 @@ def get_instances(self):
         :returns: The number of instances specified in the config, 0 if not
                   specified or if desired_state is not 'start'."""
         if self.get_desired_state() == 'start':
-            instances = self.get_min_instances()
             max_instances = self.get_max_instances()
             if max_instances is not None:
+                instances = self.get_min_instances()
```

Comment:
can you call this variable `min_instances` to make it more obvious where it came from?